x-pack/plugins/security_solution/public/flyout/right/tabs/table_tab.tsx

```diff
@@ -7,9 +7,115 @@
 
 import type { FC } from 'react';
 import React, { memo } from 'react';
-import { TimelineTabs } from '../../../../common/types';
-import { EventFieldsBrowser } from '../../../common/components/event_details/event_fields_browser';
+import { EuiPanel, EuiText } from '@elastic/eui';
+import { get } from 'lodash';
+import memoizeOne from 'memoize-one';
+import styled from 'styled-components';
+import { FieldValueCell } from '../../../common/components/event_details/table/field_value_cell';
+import type { BrowserField, BrowserFields } from '../../../../common/search_strategy';
+import type { EventFieldsData } from '../../../common/components/event_details/types';
+import { FieldNameCell } from '../../../common/components/event_details/table/field_name_cell';
+import {
+  CellActionsMode,
+  SecurityCellActions,
+  SecurityCellActionsTrigger,
+} from '../../../common/components/cell_actions';
+import { getSourcererScopeId } from '../../../helpers';
+import * as i18n from '../../../common/components/event_details/translations';
 import { useRightPanelContext } from '../context';
+import { EventFieldsBrowser } from '../../../common/components/event_details/event_fields_browser';
+import { TimelineTabs } from '../../../../common/types';
+
+const HoverActionsContainer = styled(EuiPanel)`
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  height: 25px;
+  justify-content: center;
+  left: 5px;
+  position: absolute;
+  top: -10px;
+  width: 30px;
+`;
+
+HoverActionsContainer.displayName = 'HoverActionsContainer';
+
+export const getFieldFromBrowserField = memoizeOne(
+  (keys: string[], browserFields: BrowserFields): BrowserField | undefined =>
+    get(browserFields, keys),
+  (newArgs, lastArgs) => newArgs[0].join() === lastArgs[0].join()
+);
+
+export const getColumns = ({
+  browserFields,
+  eventId,
+  contextId,
+  scopeId,
+  getLinkValue,
+  isDraggable,
+}: {
+  browserFields: BrowserFields;
+  eventId: string;
+  contextId: string;
+  scopeId: string;
+  getLinkValue: (field: string) => string | null;
+  isDraggable?: boolean;
+}) => [
+  {
+    field: 'field',
+    className: 'eventFieldsTable__fieldNameCell',
+    name: (
+      <EuiText size="xs">
+        <strong>{i18n.FIELD}</strong>
+      </EuiText>
+    ),
+    sortable: true,
+    truncateText: false,
```

Comment:
same here, removing this line and line 87 don't have an impact on the UI, we can delete them